src/style.css

```diff
@@ -1,25 +1,30 @@
 @font-face {
   font-family: "Avenir";
   src: url(./fonts/Avenir-Book.ttf);
-  font-style: normal;
   font-weight: 300;
 }
 
 @font-face {
   font-family: "Avenir";
   src: url(./fonts/Avenir-Heavy.ttf);
-  font-style: normal;
   font-weight: 800;
 }
 
+html {
+  font-family: "Avenir", sans-serif;
+}
+
+input {
```

Comment:
don't use tags in CSS, add class for inputs and use class instead of tag

Reply:
[Denchepornyuk](https://github.com/Denchepornyuk) [17 hours ago](https://github.com/mate-academy/layout_search-bar-airbnb/pull/1789#discussion_r945707970)
usually, we add box-sizing to * tag in css

so I need to use tag in this case or class for adding border-box? 
I not connect resets for all styles and I first time choose to work with block in class. 
So previously I had no tag, and I add border-box to class, and you marked it like a fault, and I changed and move it to tag... and you marked tag like a fault))) what is right?)))